wordpress-mobile/WordPress-iOS
# Stock Photos: Change "Cancel" to "Close"

Implements part of #9177 

[CAN BE MERGED]
This PR relies [on a PR](https://github.com/wordpress-mobile/MediaPicker-iOS/pull/291) to the `WPMediaPicker` project.

After that PR in `WPMediaPicker` is merged, it will be necessary to update the Podfile in this one, before merging it.

Before and after:

![cancel-close](https://user-images.githubusercontent.com/2722505/39224664-03f7f264-487b-11e8-983d-e05aa3f5eff6.jpg)


To test:
1. run `pod install` to pull the specific commit in `WPMediaPicker` necessary for this to work
2. Navigate to the Media Library
3. Tap + > Free Photo Library

The button on the top left should read "Close". Tapping it should dismiss the picker
